contracts/Joint.sol

```diff
@@ -744,20 +872,14 @@ abstract contract Joint is ySwapper {
         address[] memory swapPath,
         uint256 swapInAmount,
         uint256 minOutAmount
-    ) external onlyGovernance returns (uint256) {
-        address swapTo = swapPath[swapPath.length - 1];
-        require(swapTo == tokenA || swapTo == tokenB); // swapTo must be tokenA or tokenB
-        uint256[] memory amounts =
-            IUniswapV2Router02(router).swapExactTokensForTokens(
-                swapInAmount,
-                minOutAmount,
-                swapPath,
-                address(this),
-                now
-            );
-        return amounts[amounts.length - 1];
-    }
-
+    ) external onlyGovernance returns (uint256) {}
+
+    /*
+     * @notice
+     *  Function available to governance sweeping a specified token but tokenA and B
+     * @param expectedBalanceA, expected balance of tokenA to receive
+     * @param expectedBalanceB, expected balance of tokenB to receive
+     */
     function sweep(address _token) external onlyGovernance {
```

Comment:
less important for uniV3 but for univ2 style DEXes this would allow governance to sweep the LP token.

Reply:
Is this an issue? I would actually allow governance to sweep even tokenA and tokenB, I see it as an escape mechanism in case the strat gets bricked. What do you guys think?

Reply:
> less important for uniV3 but for univ2 style DEXes this would allow governance to sweep the LP token.

Don't we normal not protect LP tokens.

Reply:
yeah we don't but there's no point in protecting tokenA and B and not doing the same with the lp (which is the same) 

either we protect everything or we don't. 

in Strategies we have it in baseStrategy